plugins/sep/colorconfig/CustomColorOperations.hh

```diff
@@ -0,0 +1,50 @@
+//
+// Created by jelle on 02/06/2021.
+//
+
+#ifndef SCROOMCPPPLUGINS_CUSTOMCOLOROPERATIONS_HH
+#define SCROOMCPPPLUGINS_CUSTOMCOLOROPERATIONS_HH
+
+#include <boost/shared_ptr.hpp>
+#include <scroom/pipettelayeroperations.hh>
+#include <scroom/layeroperations.hh>
+#include "CustomColor.hh"
+
+
+class PipetteCommonOperationsCustomColor
+        : public PipetteLayerOperations
+                , public CommonOperations
+{
+protected:
+    int bps;
+    int spp;
+    std::vector<CustomColor*> colors;
```

Comment:
Why a `CustomColor*`, instead of just a `CustomColor` ? It'll save you some headaches trying to decide who is going to free those pointers.

If you insist on using a pointer, make sure it's a smart one.

Reply:
Changed to a smart pointer